firefox-ios/Providers/Profile.swift

```diff
@@ -693,7 +693,36 @@ open class BrowserProfile: Profile {
             if !FileManager.default.fileExists(atPath: path) {
                 try? FileManager.default.createDirectory(at: url, withIntermediateDirectories: true)
             }
-            return try RemoteSettingsService(storageDir: path, config: RemoteSettingsConfig2(server: server))
+            let encoder = JSONEncoder()
+            let appInfo = BrowserKitInformation.shared
+            let uiDevice = UIDevice.current
+            let appContext = RemoteSettingsContext(
+                appId: "org.mozilla.ios.Firefox",
+                appVersion: AppInfo.appVersion,
+                appBuild: AppInfo.buildNumber,
+                channel: appInfo.buildChannel.rawValue,
+                deviceManufacturer: "Apple",
+                deviceModel: DeviceInfo.deviceModel(),
+                osVersion: uiDevice.systemVersion,
+                locale: Locale.current.identifier,
+                os: "iOS",
+                androidSdkVersion: Build.VERSION.SDK_INT.toString(),
+                customTargetingAttributes: encoder.encode([
+                    "form_factor": switch uiDevice.userInterfaceIdiom {
+                    case .pad: "tablet"
+                    case .mac: "desktop"
+                    default: "phone"
+                    },
+                    "country": Locale.current.regionCode,
```

Comment:
Again, I'm just copying code here, but I notice that this is deprecated.  Should I be using something like `Locale.current.region.identifier` instead?

Reply:
Seems like this is code that we haven't touched for a while. Porobably safest to go with the non-deprecated version.